src/pdn/doc/add_pdn_connect.md

```diff
@@ -0,0 +1,46 @@
+## add_pdn_connect
+
+### Synopsis
+```
+  % add_pdn_connect
+    [-name name] \
```

Comment:
maybe it would be clearer if this parameter was `-grid` since it is referring to the grid defined earlier? (this would go for all the add_pdn* commands)

Reply:
Yes, I prefer -grid to -name for this situation.